### PR TITLE
Dimet: Add additional parameter for enrichment plot

### PR DESCRIPTION
**Pull request by bdartigues**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/dimet/.shed.yml [hunk 1]
```diff
@@ -2,7 +2,10 @@ categories:
 - Metabolomics
 description: DIMet is a bioinformatics pipeline for differential analysis of isotopic targeted labeling data.
 homepage_url: https://github.com/cbib/DIMet
-long_description: .
+long_description:
+  Visit our Galaxy modules:
```
Comment (bgruening) [inline tools/dimet/.shed.yml:6]: Can you put this all in yaml quotes please?

File: tools/dimet/dimet_enrichment_plot.xml [hunk 2]
```diff
@@ -29,13 +29,13 @@
                 _target_: dimet.method.MeanEnrichmentLinePlotConfig,
                 label: mean_enrichment_line_plot,
                 name: "Generate mean enrichment line plots",
-                color_lines_by: condition,
+                ###color_lines_by: condition,
```
Comment (bgruening) [inline tools/dimet/dimet_enrichment_plot.xml:32]: remove
```suggestion
```

**Review by bgruening — COMMENTED** [review]
Do you want to bump the version number to 0.2?

File: tools/dimet/dimet_enrichment_plot.xml (hunk 2 above)
Comment (bdartigues) [reply tools/dimet/dimet_enrichment_plot.xml:32]: Hi Bjorn, 
No this is minor changes. I will create a new pull request and a new fork for the tool version upgrade to 0.2.0

File: tools/dimet/.shed.yml (hunk 1 above)
Comment (bgruening) [reply tools/dimet/.shed.yml:6]: Thinking about it further, I don't think this is the correct space to advertise the tools. Please just describe here what they do.

**Comment by bgruening** [comment]
LGTM, please change the text in the shed.yml and we are ready.

File: tools/dimet/.shed.yml (hunk 1 above)
Comment (bdartigues) [reply tools/dimet/.shed.yml:6]: Hi Bjoern,

You mean all filed in the shed.yml have to be in quotes ?

File: tools/dimet/dimet_enrichment_plot.xml (hunk 2 above)
Comment (bdartigues) [reply tools/dimet/dimet_enrichment_plot.xml]: Done

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks.

**Pull request merged**